canonical/lxd-ui
# Set the correct container element to avoid unspaced content in confirmation modals

## Done

- Set the correct container element to get an appropriate spacing between the main text and the shift+click hint in confirmation modals

## QA

1. Run the LXD-UI:
    - On the demo server via the link posted by @webteam-app below. This is only available for PRs created by collaborators of the repo. Ask @lorumic or @edlerd for access.
    - With a local copy of this branch, run as described [here](../HACKING.md#setting-up-for-development).
2. Perform the following QA steps:
    - Open any of the confirmation modals.
    - Check that there is a correct spacing between the main text and the shift+click hint below it.
